docs/pages/store/reference/store-core.mdx

```diff
@@ -821,14 +823,16 @@ event Store_SpliceDynamicData(
 
 **Parameters**
 
-| Name             | Type            | Description                                                               |
-| ---------------- | --------------- | ------------------------------------------------------------------------- |
-| `tableId`        | `ResourceId`    | The ID of the table where the data is spliced.                            |
-| `keyTuple`       | `bytes32[]`     | An array representing the composite key for the record.                   |
-| `start`          | `uint48`        | The start position in bytes for the splice operation.                     |
-| `deleteCount`    | `uint40`        | The number of bytes to delete in the splice operation.                    |
-| `encodedLengths` | `PackedCounter` | The encoded lengths of the dynamic data of the record.                    |
-| `data`           | `bytes`         | The data to insert into the dynamic data of the record at the start byte. |
+| Name                | Type            | Description                                                               |
+| ------------------- | --------------- | ------------------------------------------------------------------------- |
+| `tableId`           | `ResourceId`    | The ID of the table where the data is spliced.                            |
+| `keyTuple`          | `bytes32[]`     | An array representing the composite key for the record.                   |
+| `dynamicFieldIndex` | `uint8`         | The index of the dynamic field.                                           |
+| `startWithinField`  | `uint40`        | The start byte position within the field for splicing.                    |
+| `start`             | `uint48`        | The start position in bytes for the splice operation.                     |
+| `deleteCount`       | `uint40`        | The number of bytes to delete in the splice operation.                    |
+| `encodedLengths`    | `PackedCounter` | The encoded lengths of the dynamic data of the record.                    |
+| `data`              | `bytes`         | The data to insert into the dynamic data of the record at the start byte. |
```

Comment:
Seeing this all laid out, I am wondering:
- Do we still like the `Store_SpliceDynamicData` name since splice events are now ~guaranteed to operate on one field at a time and it includes both information for splicing within the field and for all of the dynamic data? Wondering if this should be something like `Store_SpliceDynamicField` to clarify the intent.
- With above, I am also wondering if it's worth clarifying some argument names like `startWithinField` to `fieldStart` and `start` -> `dynamicDataStart`.
- "dynamic" in `dynamicFieldIndex` might be redundant since this is a `Store_SpliceDynamicData` event

Reply:
Do we need `startWithinField`? Couldn't that be computed from `dynamicFieldIndex`, `start` and `encodedLengths`? I'd prefer adding that one additional field to the event than refactoring the event name and other field names. The current naming feels very intuitive if you're thinking of the data as one byte blob, while the `Store_SpliceDynamicField` naming would be a little more intuitive if you're storing data in separate locations, but I don't there is a strong enough bias in either direction to change the name at this point.

Reply:
```ts
const fieldLengths = getFieldLengths(encodedLengths); // some sort of decoding function
const fieldStart = fieldLengths.slice(0, dynamicFieldIndex).reduce((sum, length) => sum + length, 0);
const startWithinField = start - fieldStart;
```
checks out ✅